Tests/test_SeqUtils.py

```diff
@@ -24,22 +24,25 @@ def u_crc32(seq):
     # NOTE - On Python 2 crc32 could return a signed int, but on Python 3 it is
     # always unsigned
     # Docs suggest should use crc32(x) & 0xffffffff for consistency.
-    return crc32(seq) & 0xffffffff
+    return crc32(seq) & 0xFFFFFFFF
 
 
 class SeqUtilsTests(unittest.TestCase):
-
     @classmethod
     def setUpClass(cls):
         # Example of crc64 collision from Sebastian Bassi using the
         # immunoglobulin lambda light chain variable region from Homo sapiens
         # Both sequences share the same CRC64 checksum: 44CAAD88706CC153
-        cls.str_light_chain_one = ("QSALTQPASVSGSPGQSITISCTGTSSDVGSYNLVSWYQQ"
-                                   "HPGKAPKLMIYEGSKRPSGVSNRFSGSKSGNTASLTISGL"
-                                   "QAEDEADYYCSSYAGSSTLVFGGGTKLTVL")
-        cls.str_light_chain_two = ("QSALTQPASVSGSPGQSITISCTGTSSDVGSYNLVSWYQQ"
-                                   "HPGKAPKLMIYEGSKRPSGVSNRFSGSKSGNTASLTISGL"
-                                   "QAEDEADYYCCSYAGSSTWVFGGGTKLTVL")
+        cls.str_light_chain_one = (
+            "QSALTQPASVSGSPGQSITISCTGTSSDVGSYNLVSWYQQ"
+            "HPGKAPKLMIYEGSKRPSGVSNRFSGSKSGNTASLTISGL"
+            "QAEDEADYYCSSYAGSSTLVFGGGTKLTVL"
+        )
+        cls.str_light_chain_two = (
+            "QSALTQPASVSGSPGQSITISCTGTSSDVGSYNLVSWYQQ"
+            "HPGKAPKLMIYEGSKRPSGVSNRFSGSKSGNTASLTISGL"
+            "QAEDEADYYCCSYAGSSTWVFGGGTKLTVL"
+        )
```

Comment:
Can probably use chunks of 60aa rather than 40aa, and save two lines in this section.

Reply:
Adjusted, thank you.